Ginger/Ginger/UserControlsLib/TextEditor/Common/FindReplacePage.xaml.cs

```diff
@@ -73,30 +73,44 @@ private void ReplaceClick(object sender, RoutedEventArgs e)
             string input = editor.Text.Substring(editor.SelectionStart, editor.SelectionLength);
             Match match = regex.Match(input);
             bool replaced = false;
-            if (match.Success && match.Index == 0 && match.Length == input.Length)
+            if (!editor.IsReadOnly)
```

Comment:
1- we should not allow do the replace click and then show message which it is not allowed, we need to diable the "Replace" tab from first place so user won't be able to click on it and view it in case of ReadOnly

2- you can't show specific errors like "**Raw Response** is read only text" in GENERIC page like "FindReplacePage" which used to find and replace for any content- anyway the error needs to be removed according to comment num. 1 above

Reply:
Now, if the text is read-only, the "Replace" tab is hidden to the user